.github/workflows/pr-k8s-codegen-check.yml

```diff
@@ -33,6 +33,6 @@ jobs:
         if ! git diff --exit-code --quiet; then
           echo "Changes detected:"
           git diff
-          echo "Please run './hack/update-codegen.sh' and commit the changes."
+          echo "Please run './hack/update-codegen.sh', ensuring Grafana Enterprise is not installed, and commit the changes." 
```

Comment:
why?  with the new setup... this all works out of the box 🎉 

https://github.com/grafana/grafana/blob/main/hack/update-codegen.sh#L92

Reply:
now that we are not using symlinks in enterprise, there is no need for anything special

Reply:
It didn't work for me when GE was linked. I was getting errors because of the cloudconfig dependency in GE. Does this not happen for you? Maybe my environment is messed up...